presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -71,7 +70,6 @@ private static TableDefinition partitionedTableDefinition()
 
     @Test(groups = {BASIC_SQL})
     public void testShowPartitionsFromHiveTable()
-            throws SQLException
```

Comment:
Can you remove all such things in product-tests

Reply:
i'd rather not do this this time